samples/client.conf

```diff
@@ -41,6 +41,7 @@ endpoints:
 session_retry_limit: 10
 session_retry_interval: 10000
 session_timeout: 0
+ignore_clock_skew: false
```

Comment:
Perhaps this should go on the performance section

Reply:
I don't have a strong preference, so if you prefer it to be added to the performance section I'm good with that. Shall I update that?

Reply:
Went ahead and added a commit to move the field in the performance section...